src/components/Radio/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { RadioContentProps } from './type'
+import { RadioStyle } from './style'
+
+const Radio: React.FC<RadioContentProps> = ({
+  color,
+  theme,
+  checked,
+  ...props
+}) => (
+  <RadioStyle
+    color={color}
+    theme={theme}
+    checked={checked}
+    {...props}
+    type="radio"
+  />
+)
```

Comment:
maybe thus, do more "clean code"

```suggestion
const Radio: React.FC<RadioContentProps> = ({ ...props }) => (
  <RadioStyle {...props} type="radio"/>
)
```